product_brand/reports/account_invoice_report.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2018 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import fields, models
+
+
+class AccountInvoiceReport(models.Model):
+    _inherit = "account.invoice.report"
+
+    product_brand_id = fields.Many2one(
+        comodel_name='product.brand',
+        string='Brand',
+    )
+
+    def _select(self):
+        select_str = super()._select()
+        select_str += """
+            , sub.product_brand_id as product_brand_id
+            """
+        return select_str
```

Comment:
can be summarize by

`        return super()._select() + ", sub.product_brand_id as product_brand_id"
`

Just typo, not blocking

The same below

Reply:
@bealdav Thanks for your review. I don't think we should refactor code in migration PR if it's not required. Feel free to submit a refactoring in a PR after the migration merge.